tests/Spice86.Tests/Resources/NativeDosTests/build.bat

```diff
@@ -0,0 +1,18 @@
+rem grab your Open Watcom V2 copy from https://github.com/open-watcom/open-watcom-v2/releases
```

Comment:
Can I compile this on Linux or macos ? Perhaps with dosbox staging ?

Another solution would be to include precompiled versions for unit tests in the future.

Reply:
That's good.

Yes the pre-compiled DOS binaries would be nice, but I can do that ship thoses with the DOS process managment PR later.

Reply:
ok i will add the exes later

Reply:
just ask on discord if you need help building watcom on linux - is not that complex, some minutes to get your own fresh build

Reply:
I'm on Windows so it will super easy to do with the .bat file you provided. :)

Reply:
yes just extract (not execute) the https://github.com/open-watcom/open-watcom-v2/releases/download/Current-build/open-watcom-2_0-c-win-x64.exe and fix the path in the batch